[Windows] Cannot find module './+about'

When clicking to the about route, I'm given the following error.
`EXCEPTION: Error: Uncaught (in promise): Error: Cannot find module './+about'.`

 I've tried adjust the path, and pointing directly at ./+about/index.js (& .ts) to no success.


Could this be related to the sheer mass of warnings I'm seeing in webpack & chrome: 
````
WARNING in ./~/@angular/core/src/linker/system_js_ng_module_factory_loader.js
28:15 
Critical dependency: the request of a dependency is an expression
```

--- a/webpack.config.ts
+++ b/webpack.config.ts
@@ -102,7 +102,8 @@ function webpackConfig(options: EnvOptions = {}): WebpackConfig {
     plugins: [
       // fix angular2
       new ContextReplacementPlugin(
-        /angular\/core\/(esm\/src|src)\/linker/,
+        // The (\\|\/) piece accounts for path separators in *nix and Windows
+        /angular(\\|\/)core(\\|\/)(esm(\\|\/)src|src)(\\|\/)linker/,
         root('./src'),
         resolveNgRoute(root('./src'))
       ),

